app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -377,4 +391,16 @@ def lazy_find_managed_object(managed_object)
     parent_collection = persister.vim_class_to_collection(managed_object)
     parent_collection.lazy_find(managed_object._ref)
   end
+
+  def rbvmomi_to_basic_types(obj)
```

Comment:
Ideally we could just do `obj.to_h` and the RbVmomi classes would support this, going to work on an rbvmomi PR to add this

Reply:
My change for this actually got merged but hasn't been released yet, I'll update this when a release is cut